opencti-platform/opencti-graphql/src/database/ai-llm.ts

```diff
@@ -27,7 +26,7 @@ const AI_AZURE_DEPLOYMENT = conf.get('ai:ai_azure_deployment');
 
 let client: Mistral | OpenAI | AzureOpenAI | null = null;
 let nlqChat: ChatOpenAI | ChatMistralAI | AzureChatOpenAI | null = null;
-if (AI_ENABLED && AI_TOKEN) {
+if (AI_TOKEN) {
```

Comment:
I think we need to keep the AI_ENABLED but get the value from settings instead of conf.get('ai:enabled');